x/dualstaking/keeper/delegator_reward.go

```diff
@@ -85,6 +85,14 @@ func (k Keeper) GetAllDelegatorReward(ctx sdk.Context) (list []types.DelegatorRe
 func (k Keeper) CalcRewards(stakeEntry epochstoragetypes.StakeEntry, totalReward math.Int, delegations []types.Delegation) (providerReward math.Int, delegatorsReward math.Int) {
 	effectiveDelegations, effectiveStake := k.CalcEffectiveDelegationsAndStake(stakeEntry, delegations)
 
+	// Sanity check - effectiveStake > 0
+	if effectiveStake.IsZero() {
+		utils.LavaFormatWarning("critical: Attempt to divide by zero", nil,
```

Comment:
in the warning's description (first arg) write something like "effective stake is zero" and in the error field (second arg, instead of nil) put fmt.Errorf("critical: Attempt...")

Reply:
Fixed in 20bf6f5ad2b263ea30b7641558c140ceca7a9293